### PR TITLE
serve qr code as png and improve download experience

### PR DESCRIPTION
**Pull request by salimhb**

### CODE REVIEW

File: app/controllers/owners/areas_controller.rb [hunk 1]
```diff
@@ -27,13 +27,19 @@ def show
...
           qr_code = QrCode.call(area: area)
 
-          render body: qr_code.svg
+          send_data qr_code.svg, type: 'image/svg', disposition: 'attachment', filename: "#{qr_code.file_name}.svg"
+        end
+
+        format.png do
+          qr_code = QrCode.call(area: area)
+
+          send_data qr_code.png.to_s, type: 'image/png', disposition: 'attachment', filename: "#{qr_code.file_name}.png"
```
Comment (bonflintstone) [inline app/controllers/owners/areas_controller.rb:42]: Why is it `.to_s` here but not for svgs?

Comment (salimhb) [reply app/controllers/owners/areas_controller.rb:42]: yeah, didn't like this much. `qr_code.png` is an object of class `ChunkyPNG::Image`.
I took the `.to_s` straight out of the Gem's README https://github.com/whomwah/rqrcode#as-png

Comment (bonflintstone) [reply app/controllers/owners/areas_controller.rb]: Well :shrug:

File: app/controllers/owners/areas_controller.rb [hunk 2]
```diff
@@ -25,15 +25,21 @@ def show
 
       respond_to do |format|
```
Comment (bonflintstone) [inline app/controllers/owners/areas_controller.rb:26]: How about adding here (and removing lower)

```suggestion
      qr_code = QrCode.call(area: area, format: request.format.symbol) unless request.format.symbol == :json

      respond_to do |format|
```

File: app/controllers/owners/areas_controller.rb [hunk 3]
```diff
@@ -25,15 +25,21 @@ def show
...
         format.pdf do
-          qr_code = QrCode.call(area: area)
+          qr_code = QrCode.call(area: area, format: 'pdf')
```
Comment (bonflintstone) [inline app/controllers/owners/areas_controller.rb:28]:
```suggestion
```

File: app/controllers/owners/areas_controller.rb [hunk 4]
```diff
@@ -25,15 +25,21 @@ def show
...
 
-          render pdf: qr_code.file_name, data: qr_code.data
+          render pdf: "#{qr_code.file_name}.pdf", data: qr_code.data
         end
 
         format.svg do
-          qr_code = QrCode.call(area: area)
+          qr_code = QrCode.call(area: area, format: 'svg')
```
Comment (bonflintstone) [inline app/controllers/owners/areas_controller.rb:34]:
```suggestion
```

File: app/controllers/owners/areas_controller.rb [hunk 5]
```diff
@@ -25,15 +25,21 @@ def show
...
 
-          render body: qr_code.svg
+          send_data qr_code.data, type: 'image/svg', disposition: 'attachment', filename: "#{qr_code.file_name}.svg"
+        end
+
+        format.png do
+          qr_code = QrCode.call(area: area, format: 'png')
```
Comment (bonflintstone) [inline app/controllers/owners/areas_controller.rb:40]:
```suggestion
```

**Pull request merged**